Code/Tools/AssetProcessor/native/AssetManager/LfsPointerFileValidator.h

```diff
@@ -0,0 +1,50 @@
+/*
+* Copyright (c) Contributors to the Open 3D Engine Project.
+* For complete copyright and license terms please see the LICENSE at the root of this distribution.
+*
+* SPDX-License-Identifier: Apache-2.0 OR MIT
+*
+*/
+#pragma once
+
+#include <AzCore/std/string/string.h>
+#include <AzCore/std/containers/set.h>
+
+namespace AssetProcessor
+{
+    class LfsPointerFileValidator

```

Comment:
Public comment on class responsibility (ie what is this class/what does it do)

Reply:
Will add the description.